content/refguide/published-rest-operation-parameter.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "Published REST Operation Parameter"
+parent: "published-rest-services"
+---
+
+{{% alert type="info" %}}
+
+This feature was introduced in version 7.17.0.

```

Comment:
Check with Adam to see if we can do range here. Something like 'This feature was gradually introduced throughout version 7.13 until 7.17.'

Reply:
Done